src/main/scala/com/picnicml/doddlemodel/metrics/RegressionMetrics.scala

```diff
@@ -15,4 +17,27 @@ object RegressionMetrics {
       sqrt((d.t * d) / y.length.toDouble)
     }
   }
+
+  /** Mean absolute error */
+  object Mae extends Metric {
+
+    lazy val higherValueIsBetter: Boolean = false
+
+    override def apply(y: Target, yPred: Target): Double = {
+      val d = y - yPred
+      sum(abs(d)) / y.length.toDouble
+    }
+  }
+
+
+  /** Explained variance */
+  object ExplainedVariance extends Metric {
+
+    lazy val higherValueIsBetter: Boolean = true
+
+    override def apply(y: Target, yPred: Target): Double = {
+      val d = y - yPred
+      1 - variance(d)/variance(y)
```

Comment:
Same here: `variance(y - yPred)` is better. I know I'm nitpicking, but spaces before and after `/` 😄. Another thing I do is e.g. use `1.0 - someVar` instead of `1 - someVar` (like [here](https://github.com/picnicml/doddle-model/blob/master/src/main/scala/com/picnicml/doddlemodel/linear/LogisticRegression.scala#L35)), so that it's clear immediately from that line that we are dealing with `Double`.